tests/ensure_non_running_provider.py

```diff
@@ -19,19 +19,20 @@
 # empty play to gather facts
 - hosts: all
 
-- import_playbook: "{tests_playbook}"
-  vars:
-    network_provider: initscripts
-  when:
-    - ansible_distribution_major_version != '6'
-    - network_provider_current == 'nm'
+- hosts: all
+  roles:
+    - linux-system-roles.network
+
+- hosts: all
+  tasks:
+    - debug:
+        var: network_provider_current
 
 - import_playbook: "{tests_playbook}"
   vars:
-    network_provider: nm
+    network_provider: "{{{{ 'initscripts' if network_provider_current == 'nm' else 'nm' }}}}"
```

Comment:
Why so many `{`s?

Reply:
Ansible requires two  curly braces but they are also used for str.format(), therefore they need to be escaped in Python, which requires to double them.